client/src/features/viewers/countdown/Countdown.jsx

```diff
@@ -13,19 +14,20 @@ import { fetchTimerData, sanitiseTitle, timerMessages } from './countdown.helper
 import style from './Countdown.module.scss';
 
 export default function Countdown(props) {
-  const [searchParams] = useSearchParams();
-  const { backstageEvents, time, selectedId } = props;
+  const { backstageEvents, time, selectedId, settings } = props;
   const [follow, setFollow] = useState(null);
   const [runningTimer, setRunningTimer] = useState(0);
   const [runningMessage, setRunningMessage] = useState('');
   const [delay, setDelay] = useState(0);
+  const [localTimeFormat, setLocalTimeFormat] = useState(null);
```

Comment:
I think this can be removed. The `clock`, `endTime` and `startTime` seems too verbose. Maybe we can rewrite it like this?

```js
const timeFormat = searchParams.get("format") || settings.timeFormat;

const clock = formatDate(time.clock, timeFormat);
const startTime = formatDate(follow.timeStart + delay, timeFormat);
const endTime = formatDate(follow.timeEnd + delay, timeFormat);
```

No need for hooks